crates/re_ui/src/drag_and_drop.rs

```diff
@@ -0,0 +1,309 @@
+//! Helpers for drag and drop support. Works well in combination with [`crate::list_item::ListItem`].
+
+/// Context information related to a candidate drop target, used by [`find_drop_target`] to compute the [`DropTarget`],
+/// if any.
+pub struct DropItemDescription<ItemId: Copy> {
```

Comment:
If this describes a drop target, I suggest calling it `DropTargetDescription` or `DropZoneDescription`.